UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs

```diff
@@ -17,13 +18,16 @@ public class Pipe : MonoBehaviour
 	public Pipenet pipenet;
 	public float volume = 70;
 
+	[Flags]
 	public enum Direction
 	{
-		NORTH,
-		SOUTH,
-		WEST,
-		EAST
+		NONE = 0,
+		NORTH = 1,
+		SOUTH = 2,
+		WEST = 4,
+		EAST = 8
```

Comment:
Good way of doing the values
```suggestion
		NORTH = 1 << 0,
		SOUTH = 1 << 1,
		WEST  = 1 << 2,
		EAST  = 1 << 3
```